### DISCUSSION TITLE
Design

### DISCUSSION THREAD

**Discussion by siniux** [PaulWoitaschek/Voice#1358] [Ideas]
I've been working on a different design direction for the app, while maintaining (what I think is) the main design options.
What do you think?

![1](https://user-images.githubusercontent.com/3108800/53031559-fbf4a900-3464-11e9-939c-495898c20ef2.jpg)

![2](https://user-images.githubusercontent.com/3108800/53031560-fbf4a900-3464-11e9-81bb-0f4d69a99f99.jpg)

**Comment by test2a** [comment]
Hey. 

May I ask why we have chapters there, just above the play/pause button ? Whats the point of having that and the next /previois chapter buttons?

I have had problems by way of incorrectly hitting the chapter list and going to a random position instead of pausing or playing. 

Isnt the idea of listening to an audiobook a linear affair, you start at position 00.00 and listen till end? Why should the option to jump be there?

Look at Libby or overdrive apps. They have a pause play button along with move forward or backward but no chapters. That is on the sidebar if and when you need to move to a place. Why cant we do the same, move chapter list to sidebar or near the bookmark and sleep timer?

A discussion on this would be nice

**Comment by nioncode** [comment]
I would also like to have the chapters either in a side menu or on a separate button in the toolbar. It just clutters the look imo and personally I never use it, because there are like 100 chapters in audio books and they don't have much meaning at all.

**Comment by siniux** [comment]
I don't dislike having the chapter title on the main window and the current solution of having it "clickable" makes sense. Also, with the changes of design that I applied, the click ability doesn't seem to be as problematic as it was before.

**Comment by test2a** [comment]
@Siniux how about we shift the chapter listing to sidebar and keep in its place a tooltip of sorts that only shows the current chapter name.

Essentially, keep the current look of showing the user the chapter number but shift selection to sidebar or the top where we already have a bunch of buttons.

I understand accedental selections might have reduced with your design but im just asking why keep it in the first place.

**Comment by test2a** [comment]
![Untitled Diagram](https://user-images.githubusercontent.com/39449563/55687341-62884480-5989-11e9-9057-c4775ffa1622.png)
@siniux how about this ?  I am suggesting not having the shift chapter buttons because one does not normally want to switch chapters. more discussions would be appreciated

**Comment by nioncode** [comment]
I don't use the chapter buttons at all, since they don't have much meaning in most cases (at least in my usage) and I just want to go back / forward x minutes (so I tap the rewind button a few times). I'd like to have the chapters as a separate sheet that can be swiped in from the right and not bother with them on the main screen at all.

**Comment by test2a** [comment]
@nioncode that is exactly my idea too. Chapters should not be on main listening page.

**Comment by revelous** [comment]
Personally, I appreciate having the chapter selection readily available. It makes switching back and forth from ebook to audiobook very easy.

**Comment by almondsurf** [comment]
I would love to have the ability to view ALL audio books in the initial screen. Currently one sees lists of: Current, Not started yet, and Completed.
It would be great to have a button at the top that says "Show All" that will expand to show all audiobooks on the device in one easy scrollable screen.
Maybe in the settings there could be an option to have this as a default (only because this my preference).
This screen could have a filter to show all titles listed in alphabetical order OR by % completed.
Thoughts?
